hca-dcp/submit.wdl

```diff
@@ -23,7 +24,9 @@ task get_metadata {
       "$(cut -f3 $creds)/api/workflows/v1/$(cat workflow_id.txt)/metadata" > metadata.json
```

Comment:
We can remove the api url out of the credentials file in the docker container and change this to `"https://cromwell.mint-${runtime_environment}.broadinstitute.org/api/workflows/v1/$(cat workflow_id.txt)/metadata"`. That way we can use the same docker container for both staging and dev since the url is the only difference between the two!

Reply:
This makes life much easier, we would then only need to maintain a smaller set of docker images! Will do it! Thanks